src/index.js

```diff
@@ -1,6 +1,6 @@
 const React = require('react');
 
-const CaptchaScript = (cb, hl) => { // Create script to init hCaptcha
+const CaptchaScript = (cb, hl, endpoint) => { // Create script to init hCaptcha
```

Comment:
This isn't a big deal, just a tid bit.

Reply:
agreed. I didn't wanna change the existing code but this makes sense, I'll change it.

Reply:
Yea, I belive you are correct, it would be `undefined` which means it should fail that test regardless. That's kinda why I was saying its more of syntax sugar then help.